src/main/resources/assets/languagereload/lang/pt_br.json

```diff
@@ -1,9 +1,9 @@
 {
-  "language.default.tooltip": "The default language of Minecraft",
-  "debug.reload_languages.help": "F3 + J = Recarregar idiomas (shift to cycle languages)",
+  "language.default.tooltip": "O idioma padrão do Minecraft",
+  "debug.reload_languages.help": "F3 + J = Recarregar idiomas (mudar para idiomas de ciclo)",
```

Comment:
"Shift" is a name of a key that probably shouldn't be translated in your language.